### PR TITLE
Add Rmath

### PR DESCRIPTION
**Pull request by ararslan**
Based on a combination of https://github.com/JuliaStats/Rmath-julia/blob/master/build_tarballs.jl and https://github.com/JuliaStats/RmathBuilder/blob/master/build_tarballs.jl, updated for the newer BinaryBuilder.

I didn't test it beforehand since there's CI now 😬

### CODE REVIEW

**Comment by ararslan** [comment]
I'm confused by the Azure Pipelines interface. How do I see what the problem was for the failed build?

File: R/Rmath/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+script = raw"""
+cd $WORKSPACE/srcdir/Rmath-julia-*
+make
+if [[ ${target} == *-mingw32 ]]; then
+    OUTDIR=$DESTDIR/bin
+else
+    OUTDIR=$DESTDIR/lib
+fi
+mkdir -p $OUTDIR
+mv src/libRmath-julia.* $OUTDIR
```
Comment (giordano) [inline R/Rmath/build_tarballs.jl]: Use `${libdir}` as destination so you can remove the definition of `${}` above

File: R/Rmath/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+using BinaryBuilder
+
+name = "Rmath"
+version = v"0.2.2"
+
+sources = [
+    "https://github.com/JuliaStats/Rmath-julia/archive/$(version).tar.gz" =>
```
Comment (giordano) [inline R/Rmath/build_tarballs.jl]: There is a missing v before the version number

**Comment by giordano** [comment]
> I'm confused by the Azure Pipelines interface. How do I see what the problem was for the failed build?

There's an error in the "generator" job: 
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=314&view=logs&j=96c13d9d-dfd2-51e5-2f76-4a0b3ce09abc. The URL is wrong

**Comment by ararslan** [comment]
Well that's embarrassing. Thanks, Mosè!

**Comment by ararslan** [comment]
Good to merge, or should we wait until the automated JLL registration is fixed?

**Comment by giordano** [comment]
For me it's good to go, but I'd wait for registration to be fixed before merging

**Pull request merged**